docs/app/pages/components/components-sections/tables/fixed-header-table-ng1/fixed-header-table-ng1.component.html

```diff
@@ -3,39 +3,33 @@
 <hr>
 
 <p>
-    A fixed header table is provided by the <code>scrollable-table</code> directive. The table header stays in place while
-    the table content is scrolled independently.
+    A fixed header table can be created by using the <code>fixed-header-table</code> directive. 
+    This allows the table header to remain fixed in place and visible while the table
+    content is scrolled.
 </p>
 
 <p>The attributes of this directive are:</p>
 
 <uxd-api-properties>
-    <tr uxd-api-property name="watch" type="object" binding="variable" required="true">
-        Defines the values of the rows.
+    <tr uxd-api-property name="tableHeight" required="true" type="number" binding="variable">
+        This defines the maximum height of the table.
```

Comment:
Should mention that this is in pixels. Also is it possible to use other CSS units such as vh or % for the max height? I can imagine that would be useful.